pkgs/applications/version-management/git-and-tools/hut/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildGoModule
+, fetchFromSourcehut
+, scdoc
+, unstableGitUpdater
+}:
+
+buildGoModule {
+  pname = "hut";
+  version = "unstable-2022-03-02";
+
+  src = fetchFromSourcehut {
+    owner = "~emersion";
+    repo = "hut";
+    rev = "55ad2fbd9ceeeb9e7dc203c15476fa785f1209e0";
+    sha256 = "sha256-j2IVwCm7iq3JKccPL8noRBhqw+V+4qfcpAwV65xhZk0=";
+  };
+
+  vendorSha256 = "sha256-zdQvk0M1a+Y90pnhqIpKxLJnlVJqMoSycewTep2Oux4=";
+
+  nativeBuildInputs = [
+    scdoc
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  postBuild = ''
+    make $makeFlags completions doc/hut.1
+  '';
+
+  preInstall = ''
+    make $makeFlags install
+  '';
```

Comment:
Isn't that unnescessary?

> The install phase is responsible for installing the package in the Nix store under out. The default installPhase creates the directory $out and calls make install.
> 
> https://nixos.org/manual/nixpkgs/stable/#ssec-install-phase

Reply:
No, this is `buildGoModule`, not `mkDerivation`. `buildGoModule`'s `installPhase` simply copies the binaries, and does not call make, but here we want to install the completions and manual page too